cleaners/windows_explorer.xml

```diff
@@ -18,79 +18,127 @@
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\RecentDocs"/>
-    <!-- Windows 7 jump list -->
-    <action command="delete" search="glob" path="%APPDATA%\Microsoft\Windows\Recent\AutomaticDestinations\*.automaticDestinations-ms"/>
-    <action command="delete" search="glob" path="%APPDATA%\Microsoft\Windows\Recent\CustomDestinations\*customDestinations-ms"/>
+    <!-- Windows 7 jump list: -->
+    <action command="delete" search="glob" path="%AppData%\Microsoft\Windows\Recent\AutomaticDestinations\*.automaticDestinations-ms"/>
+    <action command="delete" search="glob" path="%AppData%\Microsoft\Windows\Recent\CustomDestinations\*customDestinations-ms"/>
   </option>
   <option id="run">
     <label translators="In Windows 'Run' is the dialog in the Start menu">Run</label>
     <description>Delete the most recently used list</description>
-    <!--  run command -->
+    <!-- Run command: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\RunMRU"/>
   </option>
   <option id="search_history">
     <label>Search history</label>
     <description>Delete the search history</description>
     <action command="winreg" path="HKCU\Software\Microsoft\Search Assistant\ACMru"/>
-    <!-- Windows 10 -->
+    <!-- Windows 10: -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\WordWheelQuery"/>
   </option>
   <option id="thumbnails">
     <label>Thumbnails</label>
     <description>Delete the cache</description>
     <warning>This will restart Windows Explorer.</warning>
     <action command="process" cmd="taskkill.exe /f /IM explorer.exe"/>
-    <action command="delete" search="glob" path="%LOCALAPPDATA%\Microsoft\Windows\Explorer\thumbcache*.db"/>
+    <action command="delete" search="glob" path="%LocalAppData%\Microsoft\Windows\Explorer\thumbcache*.db"/>
     <!-- On 64-bit Windows, it is important to start the 64-bit Windows Explorer,
          so use the full path. Also, do not wait for the process to terminate.
 
          The redundant .exe is intentional to launch explorer.exe in disabled UAC
          virtualization mode rather than elevated mode because normally it
          inherits the mode from bleachbit.exe.
      -->
-    <action command="process" wait="false" cmd="%WINDIR%\explorer.exe %WINDIR%\explorer.exe"/>
+    <action command="process" wait="false" cmd="%WinDir%\explorer.exe %WINDIR%\explorer.exe"/>
+    <action command="delete" search="walk.files" path="%LocalAppData%\Microsoft\Windows\Explorer\ThumbCacheToDelete\"/>
+    <action command="delete" search="file" path="%LocalAppData%\IconCache.db"/>
+    <action command="delete" search="file" path="%LocalAppData%\IconCache.db.backup"/>
+  </option>
+  <option id="mountpoints2">
+    <label>Mount Points</label>
+    <description>Delete the data</description>
+    <warning>Use the option only if you know what you do!</warning>
```

Comment:
It isn't immediately obvious to me what this does, but it sounds scary, so maybe it shouldn't be here.

Reply:
https://www.techwalla.com/articles/how-to-delete-mountpoints2-with-regedit